src/SmartComponents/Drift/utils.js

```diff
@@ -0,0 +1,76 @@
+export function formatDate(date) {
+    const d = new Date(date);
+    return new Date(d.getTime() - d.getTimezoneOffset() * 60 * 1000).toISOString().split('T')[0];
+}
+
+;
+
+export function getDate(ago) {
+    let d = new Date();
+    d.setDate(d.getDate() - ago);
+    return formatDate(d);
+}
+
+export function parseDriftData(data) {
+    const payload = {};
+    payload.numEvents = data.meta.count;
+    payload.baselineDetail = [];
+    let unique_baselines = [];
+
+    data.data.forEach((item) => groupPayload(item.payload, payload.baselineDetail, unique_baselines));
+    payload.baselineDetail.sort(function (a, b) {
```

Comment:
We are using ES6 no need to have anonymous `function` anymore.
```suggestion
    payload.baselineDetail.sort((a, b) => {
```